django/db/models/base.py

```diff
@@ -1327,13 +1327,35 @@ def _check_managers(cls, **kwargs):
     @classmethod
     def _check_fields(cls, **kwargs):
         """Perform all field checks."""
-        errors = []
+        errors = cls._check_json_fields(**kwargs)
         for field in cls._meta.local_fields:
             errors.extend(field.check(**kwargs))
         for field in cls._meta.local_many_to_many:
             errors.extend(field.check(from_model=cls, **kwargs))
         return errors
 
+    @classmethod
+    def _check_json_fields(cls, **kwargs):
```

Comment:
This check means that **all** of the database connections must support `JSONField` if `JSONField` is used. The previous implementation allows that **at least one** connection supports it. Is this an intended change?

Reply:
Yes, all connections on which you want to create this model must support `JSONField`'s.

Reply:
Ah, I didn't catch the `router` part. I've never used it before, but if I'm correct, it should be the better option since it actually checks whether the model will actually be created on the specific connection. Cool!